dot/node.go

```diff
@@ -14,16 +14,22 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/ChainSafe/gossamer/dot/core"
+	"github.com/ChainSafe/gossamer/dot/digest"
 	"github.com/ChainSafe/gossamer/dot/network"
 	"github.com/ChainSafe/gossamer/dot/rpc"
 	"github.com/ChainSafe/gossamer/dot/state"
 	"github.com/ChainSafe/gossamer/dot/state/pruner"
+	gsync "github.com/ChainSafe/gossamer/dot/sync"
```

Comment:
nit maybe name it `dotsync` instead? `gsync` feels very nvidia-ish 😄

Reply:
Updated.